### PR TITLE
Pointed rational extension of R has uncountable extent

### PR DESCRIPTION
**Pull request by pzjp**
Added not P198 to S60. It resolves also [countable spread](https://topology.pi-base.org/properties/P000197) and [hereditarily separable](https://topology.pi-base.org/properties/P000180).
Removed redundant P27 (not [second countable](https://topology.pi-base.org/properties/P000027)) entry (cf. [this search](https://topology.pi-base.org/spaces?q=Second+countable+%2B+%7EHas+countable+extent)).

### CODE REVIEW

**Comment by yhx-12243** [comment]
P18 can also be reduced by T562.

**Comment by GeoffreySangston** [comment]
I'm still getting 

> Error SyntaxError: Unexpected token '<', "<?xml vers"... is not valid JSON

When I try to check out your branches. Is it possibly due to the use of ':' in the branch name?

**Comment by prabau** [comment]
@GeoffreySangston It's probably because the repository for this PR is not the official pi-base repository.  For new contributors we ask them to do a few PRs from their own repository first to make sure things are ok.

There is a cloning procedure explained in https://github.com/pi-base/data/wiki/Reviewing to make a preview available for the PR.  It's a little cumbersome but can be used.  I'll clone this particular branch to see.

**Comment by prabau** [comment]
You can now do a preview from branch `pr1148` in the official repo.

**Comment by prabau** [comment]
@pzjp FYI, to check the preview of the branch, go to the Advanced tab in the pi-base web page and change the branch name as needed.  Sometimes it does not work, and you have to first click Reset and then change the branch name again.

**Comment by prabau** [comment]
@pzjp The change itself looks fine.  The preview mode shows that the deleted traits are still derivable.

But looking at the description of the space, it seems it does not quite match what's in Steen & Seebach.  The description here is not incorrect per se, but there is no need to include the Euclidean open sets.  The sets $\\{x\\} \cup (D \cap U)$ are enough to generate the topology.

Would you be able to make a change for that?  (We could do it in a separate PR, but we may as well take the opportunity to do it here if it's simple to do.)

**Comment by prabau** [comment]
@pzjp FYI, I forgot if I mentioned it to you before.  But if not, there are two ways to preview things when making changes in some file.

(1) In each editor window in vscode/github.dev, there is a button on the upper right to display the contents of the file in preview mode.

(2) One can cut and paste the text into the "Editor Preview" window (https://topology.pi-base.org/dev/preview), also accessible from the Advanced tab.  That is convenient to show the expansion of pi-base shortcuts like `{S60}`, etc.

**Comment by pzjp** [comment]
Added also [locally countable](https://topology.pi-base.org/properties/P000093).

Among interesting missing traits of S60 there is [homogeneity](https://topology.pi-base.org/properties/P000086) (!). See the note on [StackExchange](https://math.stackexchange.com/questions/5013911).

File: spaces/S000060/properties/P000093.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+space: S000060
+property: P000093
+value: true
+refs:
+- doi: 10.1007/978-1-4612-6290-9_6
+  name: Counterexamples in Topology
+---
+
+By definition, base of topology consists of countable sets $\{x\}\cup((x-\varepsilon,x+\varepsilon)\cap\mathbb Q)$ (for $x\in\mathbb R$, $\varepsilon>0$).
```
Comment (prabau) [inline spaces/S000060/properties/P000093.md:10]:
```suggestion
By definition, the topology has a base consisting of countable sets $\{x\}\cup((x-\varepsilon,x+\varepsilon)\cap\mathbb Q)$ (for $x\in\mathbb R$, $\varepsilon>0$).
```
minor grammar fix

**Comment by prabau** [comment]
Interesting mathse post about homogeneity.  I need to read it.

For now I'll approve this PR.

**Comment by prabau** [comment]
@StevenClontz Piotr has done great contributions so far without any issues.  Can you give him write access to the official repository?

**Review by prabau — APPROVED** [review]

**Pull request merged**